start.bash

```diff
@@ -49,15 +71,17 @@ else
     fi
 fi
 
+wait_until "minikube" "minikube status" "apiserver: Running"
+
 # Test if a Kubernetes resource is alive
 # "$1" shoulde be namespace id e.g. argo
 # "$2" should be resource id e.g. pod/argo-server
 function is_resource_alive() {
     local type=$(echo "$2" | cut -d / -f1)
     local name=$(echo "$2" | cut -d / -f2)
-    if kubectl get -n "$1" "$2" | grep -q -o "$name" >/dev/null; then
+    if kubectl get -n "$1" "$2" 2>/dev/null | grep -q -o "$name" >/dev/null; then
```

Comment:
Can use `kubectl get pod mysql -o jsonpath='{.status.containerStatuses[0].ready}'` to get Pod status.